### ISSUE TITLE
scan fails when test targets have spaces in their names

### ISSUE THREAD

**Issue by zioyero** [lyndsey-ferguson/fastlane-plugin-test_center#93]
### New Issue Checklist

- [X] Updated `fastlane-plugin-test_center` to the latest version
- [ X] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [X ] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [X] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

### Issue Description

After updating to fastlane v2.114.0, `multi_scan` started failing to execute my tests. All else being equal, it works with version 2.113.0 and fails with 2.114.0. (It also fails with 2.115.0). 

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre>+--------------------------------------+---------+--------------------------------------------------------------------+
|                                                    [32mUsed plugins[0m                                                     |
+--------------------------------------+---------+--------------------------------------------------------------------+
| Plugin                               | Version | Action                                                             |
+--------------------------------------+---------+--------------------------------------------------------------------+
| fastlane-plugin-test_center          | 3.6.0   | suppressed_tests suppress_tests suppress_tests_from_junit          |
|                                      |         | tests_from_xctestrun collate_test_result_bundles tests_from_junit  |
|                                      |         | multi_scan collate_html_reports collate_junit_reports              |
|                                      |         | collate_json_reports                                               |
| fastlane-plugin-git_tags             | 0.1.1   | git_tags                                                           |
| fastlane-plugin-set_jira_fix_version | 1.0.0   | set_jira_fix_version jira_issue_keys_from_changelog                |
+--------------------------------------+---------+--------------------------------------------------------------------+

[23:39:14]: [32m----------------------------------------[0m
[23:39:14]: [32m--- Step: Verifying fastlane version ---[0m
[23:39:14]: [32m----------------------------------------[0m
[23:39:14]: Your fastlane version 2.115.0 matches the minimum requirement of 2.95.0  ✅
[23:39:15]: [32m------------------------------[0m
[23:39:15]: [32m--- Step: default_platform ---[0m
[23:39:15]: [32m------------------------------[0m
[23:39:15]: [32mDriving the lane 'ios quick_test' 🚀[0m
[23:39:15]: [32m-----------------------------[0m
[23:39:15]: [32m--- Step: setup_circle_ci ---[0m
[23:39:15]: [32m-----------------------------[0m
[23:39:15]: Not running on CI, skipping `setup_circle_ci`
[23:39:15]: [32m------------------------[0m
[23:39:15]: [32m--- Step: multi_scan ---[0m
[23:39:15]: [32m------------------------[0m

+----------------------+----------------------+
| [32mSummary for multi_scan (test_center v3.6.0)[0m |
+----------------------+----------------------+
| try_count            | 3                    |
+----------------------+----------------------+

[23:39:15]: [36m$ xcodebuild -showBuildSettings -workspace [REDACTED].xcworkspace -scheme App\ Unit\ Tests[0m
[23:39:19]: [36m$ xcodebuild -showBuildSettings -workspace [REDACTED].xcworkspace -scheme App\ Unit\ Tests[0m

+------------------------------------------------+----------------------------------------------------------------------------------------+
|                                                        [32mSummary for scan 2.115.0[0m                                                         |
+------------------------------------------------+----------------------------------------------------------------------------------------+
| workspace                                      | [REDACTED].xcworkspace                                                                     |
| devices                                        | ["iPhone X"]                                                                           |
| scheme                                         | App Unit Tests                                                                         |
| clean                                          | false                                                                                  |
| output_directory                               | test_output/unit_tests/                                                                |
| skip_slack                                     | true                                                                                   |
| build_for_testing                              | true                                                                                   |
| derived_data_path                              | /Users/zioyero/Library/Developer/Xcode/DerivedData/[REDACTED]-avwymqhwchacurdvsnnpvlafsdne |
| skip_detect_devices                            | false                                                                                  |
| reset_simulator                                | false                                                                                  |
| reinstall_app                                  | false                                                                                  |
| app_identifier                                 | com.[REDACTED].[REDACTED]                                                                     |
| open_report                                    | false                                                                                  |
| output_types                                   | html,junit                                                                             |
| buildlog_path                                  | ~/Library/Logs/scan                                                                    |
| include_simulator_logs                         | false                                                                                  |
| should_zip_build_products                      | false                                                                                  |
| result_bundle                                  | false                                                                                  |
| use_clang_report_name                          | false                                                                                  |
| disable_concurrent_testing                     | false                                                                                  |
| skip_build                                     | false                                                                                  |
| slack_use_webhook_configured_username_and_icon | false                                                                                  |
| slack_username                                 | fastlane                                                                               |
| slack_icon_url                                 | https://s3-eu-west-1.amazonaws.com/fastlane.tools/fastlane.png                         |
| slack_only_on_failure                          | false                                                                                  |
| fail_build                                     | true                                                                                   |
| xcode_path                                     | /Applications/Xcode.app                                                                |
+------------------------------------------------+----------------------------------------------------------------------------------------+

[23:39:22]: [36m$ set -o pipefail && env NSUnbufferedIO=YES xcodebuild -workspace [REDACTED].xcworkspace -scheme App\ Unit\ Tests -destination 'platform=iOS Simulator,id=FDEC0F6E-C469-4C87-8BF9-7734DFB9E4CC' -derivedDataPath '/Users/zioyero/Library/Developer/Xcode/DerivedData/[REDACTED]-avwymqhwchacurdvsnnpvlafsdne' build-for-testing | tee '/Users/zioyero/Library/Logs/scan/App Unit Tests-App Unit Tests.log' | xcpretty  --report html --output '/Users/zioyero/dev/[REDACTED]-ios/test_output/unit_tests/report.html' --report junit --output '/Users/zioyero/dev/[REDACTED]-ios/test_output/unit_tests/report.junit' --report junit --output '/var/folders/0y/vxfg4pzd3r17dm0kb04qp32m0000gn/T/junit_report20190205-97054-1k43m8c' [0m
[23:39:22]: ▸ [35mLoading...[0m
[23:39:27]: ▸ [35m[33mThe iOS Simulator deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.1. (in target 'AFNetworking')[0m
[23:39:27]: ▸ [35m[33mThe iOS Simulator deployment target is set to 5.0, but the range of supported deployment target versions for this platform is 8.0 to 12.1. (in target 'SAMKeychain-iOS')[0m
[23:39:27]: ▸ [35m[33mThe iOS Simulator deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.1. (in target '[REDACTED]-iOS')[0m
[23:39:27]: ▸ [35m[33mThe iOS Simulator deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.1. (in target '[REDACTED]-iOS')[0m
[23:39:27]: ▸ [35m[33mThe iOS Simulator deployment target is set to 6.0, but the range of supported deployment target versions for this platform is 8.0 to 12.1. (in target 'CocoaLumberjack-iOS')[0m
[23:39:27]: ▸ [35m[33mThe iOS Simulator deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.1. (in target '[REDACTED]KeypadComm')[0m
[23:39:27]: ▸ [35m[33mThe iOS Simulator deployment target is set to 7.0, but the range of supported deployment target versions for this platform is 8.0 to 12.1. (in target '[REDACTED]')[0m
[23:39:27]: ▸ [35m[33mThe iOS Simulator deployment target is set to 6.0, but the range of supported deployment target versions for this platform is 8.0 to 12.1. (in target 'SocketRocket')[0m
[23:39:27]: ▸ [35m[33mThe iOS Simulator deployment target is set to 6.1, but the range of supported deployment target versions for this platform is 8.0 to 12.1. (in target 'MQTTClient')[0m
[23:39:27]: ▸ [35m[33mThe iOS Simulator deployment target is set to 4.3, but the range of supported deployment target versions for this platform is 8.0 to 12.1. (in target 'MXLCalendarManager')[0m
[23:39:27]: ▸ [35m[33mThe iOS Simulator deployment target is set to 4.3, but the range of supported deployment target versions for this platform is 8.0 to 12.1. (in target 'NUIParse')[0m
[23:39:27]: ▸ [35m[33mThe iOS Simulator deployment target is set to 5.1, but the range of supported deployment target versions for this platform is 8.0 to 12.1. (in target 'NUI')[0m
[23:39:27]: ▸ [35m[33mThe iOS Simulator deployment target is set to 4.3, but the range of supported deployment target versions for this platform is 8.0 to 12.1. (in target 'libextobjc')[0m
[23:39:27]: ▸ [35m[33mSkipping duplicate build file in Compile Sources build phase: /Users/zioyero/dev/[REDACTED]-ios/App/Model/Business/Video/Streaming/Call Record/AGDoorbell+AGCallRecord.m (in target '[REDACTED]')[0m
[23:39:27]: [35m[33m▸[0m [39;1mProcessing[0m Info.plist[0m
[23:39:27]: [35m[33m▸[0m [39;1mLinking[0m AGGenericMediaNotificationServiceExtension[0m
[23:39:28]: [35m[33m▸[0m [39;1mLinking[0m [REDACTED]\[0m
[23:39:28]: ▸ [35m[33m⚠️  ld: directory not found for option '-L/Users/zioyero/dev/[REDACTED]-ios/Sezam/GoogleAnalytics'[0m
[23:39:28]: [35m[33m▸[0m [39;1mLinking[0m AGDoorbellNotificationContentExtension[0m
[23:39:28]: [35m[33m▸[0m [39;1mProcessing[0m Info.plist[0m
[23:39:28]: [35m[33m▸[0m [39;1mLinking[0m AG[REDACTED]ExitNotificationContentExtension[0m
[23:39:28]: [35m[33m▸[0m [39;1mLinking[0m [REDACTED]WatchApp\[0m
[23:39:28]: [35m[33m▸[0m [39;1mProcessing[0m Info.plist[0m
[23:39:28]: [35m[33m▸[0m [39;1mProcessing[0m Info.plist[0m
[23:39:28]: [35m[33m▸[0m [39;1mGenerating '[REDACTED]\ Today.appex.dSYM'[0m
[23:39:28]: [35m[33m▸[0m [39;1mProcessing[0m Info.plist[0m
[23:39:28]: [35m[33m▸[0m [39;1mProcessing[0m Info.plist[0m
[23:39:28]: [35m[33m▸[0m [39;1mCopying[0m /Users/zioyero/Library/Developer/Xcode/DerivedData/[REDACTED]-avwymqhwchacurdvsnnpvlafsdne/Build/Products/[REDACTED]\ Development-watchsimulator/[REDACTED]WatchApp\ Extension.appex[0m
[23:39:28]: [35m[33m▸[0m [39;1mLinking[0m [REDACTED][0m
[23:39:28]: ▸ [35m[33m⚠️  ld: directory not found for option '-L/Users/zioyero/dev/[REDACTED]-ios/Sezam/GoogleAnalytics'[0m
[23:39:31]: [35m[33m▸[0m [39;1mRunning script[0m 'Fabric Init'[0m
[23:39:31]: [35m[33m▸[0m [39;1mProcessing[0m [REDACTED]-Info.plist[0m
[23:39:31]: [35m[33m▸[0m [39;1mGenerating '[REDACTED].app.dSYM'[0m
[23:39:33]: [35m[33m▸[0m [39;1mCopying[0m /Users/zioyero/Library/Developer/Xcode/DerivedData/[REDACTED]-avwymqhwchacurdvsnnpvlafsdne/Build/Products/[REDACTED]\ Development-iphonesimulator/[REDACTED]\ Today.appex[0m
[23:39:33]: [35m[33m▸[0m [39;1mCopying[0m /Users/zioyero/Library/Developer/Xcode/DerivedData/[REDACTED]-avwymqhwchacurdvsnnpvlafsdne/Build/Products/[REDACTED]\ Development-iphonesimulator/AGGenericMediaNotificationServiceExtension.appex[0m
[23:39:33]: [35m[33m▸[0m [39;1mCopying[0m /Users/zioyero/Library/Developer/Xcode/DerivedData/[REDACTED]-avwymqhwchacurdvsnnpvlafsdne/Build/Products/[REDACTED]\ Development-iphonesimulator/AGDoorbellNotificationContentExtension.appex[0m
[23:39:33]: [35m[33m▸[0m [39;1mCopying[0m /Users/zioyero/Library/Developer/Xcode/DerivedData/[REDACTED]-avwymqhwchacurdvsnnpvlafsdne/Build/Products/[REDACTED]\ Development-iphonesimulator/[REDACTED]ContentExtension.appex[0m
[23:39:33]: [35m[33m▸[0m [39;1mCopying[0m /Users/zioyero/Library/Developer/Xcode/DerivedData/[REDACTED]-avwymqhwchacurdvsnnpvlafsdne/Build/Products/[REDACTED]\ Development-watchsimulator/[REDACTED]WatchApp.app[0m
[23:39:33]: [35m[33m▸[0m [39;1mProcessing[0m Unit-Tests-Info.plist[0m
[23:39:33]: [35m[33m▸[0m [39;1mLinking[0m App\[0m
[23:39:35]: [35m[33m▸[0m [39;1mTest build[0m Succeeded[0m
+--------------------+---+
|      Test Results      |
+--------------------+---+
| Number of tests    | 0 |
| Number of failures | [32m0[0m |
+--------------------+---+

[23:39:36]: [32m------------------------------------------------------[0m
[23:39:36]: [32m--- Starting test run on testable 'App Unit Tests' ---[0m
[23:39:36]: [32m------------------------------------------------------[0m
[23:39:37]: [36m$ xcodebuild -showBuildSettings -workspace [REDACTED].xcworkspace -scheme App\ Unit\ Tests[0m
[23:39:40]: [33mCommand timed out after 3 seconds on try 1 of 4, trying again with a 6 second timeout...[0m
[23:39:44]: [36m$ xcodebuild -showBuildSettings -workspace [REDACTED].xcworkspace -scheme App\ Unit\ Tests[0m
[23:39:47]: [33mCommand timed out after 3 seconds on try 1 of 4, trying again with a 6 second timeout...[0m

+------------------------------------------------+-------------------------------------------------------------------------------------------------------+
|                                                                [32mSummary for scan 2.115.0[0m                                                                |
+------------------------------------------------+-------------------------------------------------------------------------------------------------------+
| workspace                                      | [REDACTED].xcworkspace                                                                                    |
| devices                                        | ["iPhone X"]                                                                                          |
| scheme                                         | App Unit Tests                                                                                        |
| skip_slack                                     | true                                                                                                  |
| test_without_building                          | true                                                                                                  |
| derived_data_path                              | /Users/zioyero/Library/Developer/Xcode/DerivedData/[REDACTED]-avwymqhwchacurdvsnnpvlafsdne                |
| skip_detect_devices                            | false                                                                                                 |
| reset_simulator                                | false                                                                                                 |
| reinstall_app                                  | false                                                                                                 |
| app_identifier                                 | com.[REDACTED].iossapp                                                                                    |
| open_report                                    | false                                                                                                 |
| buildlog_path                                  | ~/Library/Logs/scan                                                                                   |
| include_simulator_logs                         | false                                                                                                 |
| should_zip_build_products                      | false                                                                                                 |
| result_bundle                                  | false                                                                                                 |
| use_clang_report_name                          | false                                                                                                 |
| disable_concurrent_testing                     | false                                                                                                 |
| skip_build                                     | false                                                                                                 |
| slack_use_webhook_configured_username_and_icon | false                                                                                                 |
| slack_username                                 | fastlane                                                                                              |
| slack_icon_url                                 | https://s3-eu-west-1.amazonaws.com/fastlane.tools/fastlane.png                                        |
| slack_only_on_failure                          | false                                                                                                 |
| clean                                          | false                                                                                                 |
| only_testing                                   | [REDACTED]                                        |
| output_directory                               | test_output/unit_tests/results-App Unit Tests                                                         |
| output_types                                   | html,junit                                                                                            |
| output_files                                   | report.html,report.junit                                                                              |
| fail_build                                     | true                                                                                                  |
| xcode_path                                     | /Applications/Xcode.app                                                                               |
+------------------------------------------------+-------------------------------------------------------------------------------------------------------+

[23:39:53]: [36m$ set -o pipefail && env NSUnbufferedIO=YES xcodebuild -workspace [REDACTED].xcworkspace -scheme App\ Unit\ Tests -destination 'platform=iOS Simulator,id=FDEC0F6E-C469-4C87-8BF9-7734DFB9E4CC' -derivedDataPath '/Users/zioyero/Library/Developer/Xcode/DerivedData/[REDACTED]-avwymqhwchacurdvsnnpvlafsdne' -only-testing:[REDACTED] test-without-building | tee '/Users/zioyero/Library/Logs/scan/App Unit Tests-App Unit Tests.log' | xcpretty  --report html --output '/Users/zioyero/dev/[REDACTED]-ios/test_output/unit_tests/results-App Unit Tests/report.html' --report junit --output '/Users/zioyero/dev/[REDACTED]-ios/test_output/unit_tests/results-App Unit Tests/report.junit' --report junit --output '/var/folders/0y/vxfg4pzd3r17dm0kb04qp32m0000gn/T/junit_report20190205-97054-16q3c7j' [0m
[23:39:53]: ▸ [35mLoading...[0m
[23:39:55]: ▸ [35mxcodebuild: error: Failed to build workspace [REDACTED] with scheme App Unit Tests.[0m
[23:39:55]: ▸ [35mReason: Tests in the target “App\ Unit\ Tests” can’t be run because “App\ Unit\ Tests” isn’t a member of the specified scheme.[0m
xcodebuild: error: Failed to build workspace [REDACTED] with scheme App Unit Tests.
Reason: Tests in the target “App\ Unit\ Tests” can’t be run because “App\ Unit\ Tests” isn’t a member of the specified scheme.
[23:39:55]: [31mExit status: 70[0m
+----------------------------+----------------------------------------------------------------------------------------+
|                                                    [33mLane Context[0m                                                     |
+----------------------------+----------------------------------------------------------------------------------------+
| DEFAULT_PLATFORM           | ios                                                                                    |
| PLATFORM_NAME              | ios                                                                                    |
| LANE_NAME                  | ios quick_test                                                                         |
| SCAN_DERIVED_DATA_PATH     | /Users/zioyero/Library/Developer/Xcode/DerivedData/[REDACTED]-avwymqhwchacurdvsnnpvlafsdne |
| SCAN_GENERATED_PLIST_FILES | []                                                                                     |
| SCAN_GENERATED_PLIST_FILE  |                                                                                        |
+----------------------------+----------------------------------------------------------------------------------------+
[23:39:55]: [31mError building/testing the application - see the log above[0m

+------+----------------------------+-------------+
|                [32mfastlane summary[0m                 |
+------+----------------------------+-------------+
| Step | Action                     | Time (in s) |
+------+----------------------------+-------------+
| 1    | Verifying fastlane version | 0           |
| 2    | default_platform           | 0           |
| 3    | setup_circle_ci            | 0           |
| 💥   | [31mmulti_scan[0m                 | 40          |
+------+----------------------------+-------------+

[23:39:55]: [31mfastlane finished with errors[0m
</pre>
</details>

### Environment

<details>
  <pre><details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.14                                       |
| Ruby                        | 2.3.8                                       |
| Bundler?                    | true                                        |
| Git                         | git version 2.17.2 (Apple Git-113)          |
| Installation Source         | ~/.rvm/gems/ruby-2.3.8/bin/fastlane         |
| Host                        | Mac OS X 10.14 (18A391)                     |
| Ruby Lib Dir                | ~/.rvm/rubies/ruby-2.3.8/lib                |
| OpenSSL Version             | OpenSSL 1.0.2p  14 Aug 2018                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 10.1                                        |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
lane :quick_test do |options|
  multi_scan(workspace: "[REDACTED].xcworkspace", devices: ["iPhone X"], scheme: "App Unit Tests", clean: false, output_directory: "test_output/unit_tests/", try_count: 3, skip_slack: true)
end
```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby

# you can even provide different app identifiers, Apple IDs and team names per lane:
# More information: https://docs.fastlane.tools/advanced/#appfile

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.115.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                               | Version | Update-Status |
| ------------------------------------ | ------- | ------------- |
| fastlane-plugin-test_center          | 3.6.0   | ✅ Up-To-Date  |
| fastlane-plugin-git_tags             | 0.1.1   | ✅ Up-To-Date  |
| fastlane-plugin-set_jira_fix_version | 1.0.0   | ✅ Up-To-Date  |

<details><summary><b>Loaded gems</b></summary>

| Gem                                  | Version      |
| ------------------------------------ | ------------ |
| did_you_mean                         | 1.0.0        |
| bundler                              | 2.0.1        |
| rake                                 | 12.3.2       |
| CFPropertyList                       | 3.0.0        |
| concurrent-ruby                      | 1.1.4        |
| i18n                                 | 1.5.1        |
| minitest                             | 5.11.3       |
| thread_safe                          | 0.3.6        |
| tzinfo                               | 1.2.5        |
| activesupport                        | 5.2.2        |
| public_suffix                        | 2.0.5        |
| addressable                          | 2.6.0        |
| atomos                               | 0.1.3        |
| babosa                               | 1.0.2        |
| claide                               | 1.0.2        |
| colored                              | 1.2          |
| colored2                             | 3.1.2        |
| highline                             | 1.7.10       |
| commander-fastlane                   | 4.4.6        |
| declarative                          | 0.0.10       |
| declarative-option                   | 0.1.0        |
| digest-crc                           | 0.4.1        |
| unf_ext                              | 0.0.7.5      |
| unf                                  | 0.1.4        |
| domain_name                          | 0.5.20180417 |
| dotenv                               | 2.6.0        |
| emoji_regex                          | 1.0.1        |
| excon                                | 0.62.0       |
| multipart-post                       | 2.0.0        |
| faraday                              | 0.15.4       |
| http-cookie                          | 1.0.3        |
| faraday-cookie_jar                   | 0.0.6        |
| faraday_middleware                   | 0.13.1       |
| fastimage                            | 2.1.5        |
| gh_inspector                         | 1.1.3        |
| jwt                                  | 2.1.0        |
| memoist                              | 0.16.0       |
| multi_json                           | 1.13.1       |
| os                                   | 1.0.0        |
| signet                               | 0.11.0       |
| googleauth                           | 0.6.7        |
| httpclient                           | 2.8.3        |
| mime-types-data                      | 3.2018.0812  |
| mime-types                           | 3.2.2        |
| uber                                 | 0.1.0        |
| representable                        | 3.0.4        |
| retriable                            | 3.1.2        |
| google-api-client                    | 0.23.9       |
| google-cloud-env                     | 1.0.5        |
| google-cloud-core                    | 1.3.0        |
| google-cloud-storage                 | 1.16.0       |
| json                                 | 2.1.0        |
| mini_magick                          | 4.5.1        |
| multi_xml                            | 0.6.0        |
| plist                                | 3.5.0        |
| rubyzip                              | 1.2.2        |
| security                             | 0.1.3        |
| naturally                            | 2.2.0        |
| simctl                               | 1.6.5        |
| slack-notifier                       | 2.3.2        |
| terminal-notifier                    | 1.8.0        |
| unicode-display_width                | 1.4.1        |
| terminal-table                       | 1.8.0        |
| tty-screen                           | 0.6.5        |
| tty-cursor                           | 0.6.0        |
| tty-spinner                          | 0.9.0        |
| word_wrap                            | 1.0.0        |
| nanaimo                              | 0.2.6        |
| xcodeproj                            | 1.8.0        |
| rouge                                | 2.0.7        |
| xcpretty                             | 0.3.0        |
| xcpretty-travis-formatter            | 1.0.0        |
| fastlane-plugin-git_tags             | 0.1.1        |
| oauth                                | 0.5.4        |
| jira-ruby                            | 1.2.0        |
| fastlane-plugin-set_jira_fix_version | 1.0.0        |
| xctest_list                          | 1.1.6        |
| fastlane-plugin-test_center          | 3.6.0        |
| netrc                                | 0.11.0       |
| rest-client                          | 2.0.2        |
| semantic                             | 1.6.1        |
</details>

*generated on:* **2019-02-05**
</details></pre>
</details>

**Comment by lyndsey-ferguson** [comment]
Ok, so I updated on my local and ran my examples: no problems.

My guess is that `only_testing                                   | [REDACTED]` is running tests that are for a "test target", `App Unit Tests`. But I cannot confirm.

So:
1. are the tests in `only_testing` in the test target `App Unit Tests`?
2. is that test target part of the `App Unit Tests` Scheme?
3. Can you provide a sample project that exhibits this same issue?

**Comment by quintuschen** [comment]
I've met same problem as well. The problem for me is our scheme name has white spaces. For example, "UI Tests", and when we give command `bundle exec fastlane run multi_scan 'scheme:UI Tests' ...` it will parse the scheme as `UI\\\ Tests` with fastlane v2.114 and onwards.

**Comment by lyndsey-ferguson** [comment]
Oh, this definitely could be the problem with this particular example as it too has spaces. Interesting.

**Comment by lyndsey-ferguson** [comment]
@zioyero can you remove the spaces from the Scheme to test? If it works, that is your work-around for the moment. I will come up with a fix if this is the case.

**Comment by zioyero** [comment]
I was able to implement a workaround, but it required removing whitespaces not only from the Scheme name but also from the test target name. Once neither had any whitespace, everything worked as expected.

**Comment by MarshalGeazipp** [comment]
Hi, 
I have the same problem. My scheme name also contains a whitespace. 
I took a look at the changes between fastlane 2.113.0 and 2.114.0 and I found this commit that looks suspicious: https://github.com/fastlane/fastlane/commit/0307f12a7021ba9436afb9a328a5ec08bcbcc175
Maybe this can help you...

**Comment by MarshalGeazipp** [comment]
So my workaround is setting the fastlane version to 2.113.0 in my Gemfile like:
gem 'fastlane', '2.113.0'

**Comment by lyndsey-ferguson** [comment]
@MarshalGeazipp do you have an example of what is actually sent to `xcodebuild` on the command line? What the option is for only-testing is when sent to `scan`?

**Comment by lyndsey-ferguson** [comment]
@MarshalGeazipp don't worry, it looks like you tracked down the change that causes this to break. Two shellescapes does not equal one shellescape

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.6.3](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.6.3)
